projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -223,6 +223,9 @@ public function export_to_gdrive() {
 
 		$sheet = Google_Drive::create_sheet( $user_id, $spreadsheet_title, $sheet_data );
 
+		$tracking = new Automattic\Jetpack\Tracking();
```

Comment:
As this is a namespaced class/file, please use root path for the class name, something similar was done on https://github.com/Automattic/jetpack/pull/29102:

```php
$tracking = new \Automattic\Jetpack\Tracking();
```

Reply:
Thanks, just fixed it!